.npmignore

```diff
@@ -61,8 +61,16 @@ dist/**
 changelog.md
 
 # dev files
+.codecov.yml
+.cz-config.js
 .DS_Store
 .editorconfig
 .eslintignore
 .eslintrc.cjs
+.gitattributes
+.mailmap
 *.tgz
+CODE_OF_CONDUCT.md
+eslint-local-rules.cjs
+vercel.json
+yarn.lock
```

Comment:
is this lock used by npm consumers? i never understood that bit.

otherwise this lgtm

Reply:
nope